src/Material/Lists.elm

```diff
@@ -0,0 +1,174 @@
+module Material.Lists
+    exposing
+        ( ul
+        , li
+        , li2
+        , li3
+        , primaryContent
+        , avatar
+        , icon
+        , secondaryContent
+        , secondaryInfo
+        , secondaryAction
+        , textBody
+        , subTitle
+        )
+
+{-| From the [Material Design Lite documentation](http://www.getmdl.io/components/index.html#lists-section):
+
+> Lists present multiple line items vertically as a single continuous element.
+> Refer the Material Design Spec to know more about the content options.
+
+See also the
+[Material Design Specification]([https://material.google.com/components/lists.html).
+
+Refer to [this site](http://debois.github.io/elm-mdl#/lists)
+for a live demo and example code.
+
+# Main list elements
+@docs ul, li, li2, li3
+
+# Content configuration for one or multiple lines
+@docs primaryContent, avatar, icon
+
+# Content configuration for multiple lines
+@docs secondaryContent, secondaryInfo, secondaryAction, textBody, subTitle
+-}
+
+import Platform.Cmd exposing (Cmd, none)
+import Html.Events as Html
+import Html exposing (Html, Attribute)
+import Parts exposing (Indexed)
+import Material.Options as Options exposing (Property, Style, cs, nop)
+
+
+{-| Main list function
+-}
+ul :
+    List (Property {} m)
+    -> List (Html m)
+    -> Html m
+ul options nodes =
```

Comment:
Simpler implementation of the same thing:

   ul options = 
      Options.styled Html.ul (cs "mdl-list ++ options)